test/flaskphiid/test_annotation.py

```diff
@@ -369,13 +369,32 @@ def test_split_annotations_by_subtypes_multi_type_type_exception(self):
         anns += [AnnotationFactory.from_hutchner(self.sample_hutchner[6])]
         sorted_anns = sorted(anns, key=lambda x: x.start)
         merged = MergedAnnotation()
-        for ann in anns:
+        for ann in sorted_anns:
             merged.add_annotation(ann)
         self.assertEqual(merged.type, "UNKNOWN")
         with self.assertRaises(IncompatibleTypeException) as context:
             actual = merged.split_annotations_by_subtypes()
         self.assertTrue("URL_OR_IP" in context.exception.type_set)
         self.assertTrue("NAME" in context.exception.type_set)
 
+    def test_split_annotations_by_subtypes_single_type_no_split(self):
```

Comment:
ha. that's a serious function name.
so this means - if there are three side by side tokens, with the same label, make them a single annotation?

Reply:
yep. basically to ensure that in the case of side-by-side, same-type tokens, we get a merged annotation back.